File: 3-Strings/1168-LED/1168.py
for _ in range(int(input())):
    n = input()
    n = [int(i) for i in n]
    cont = 0
    for i in n:
        if i == 1:
            cont += 2
        elif i == 2 or i == 3 or i == 5:
            cont += 5
        elif i == 4:
            cont += 4
        elif i == 6 or i == 9 or i == 0:
            cont += 6
        elif i == 7:
            cont += 3
        elif i == 8:
            cont += 7
    print (cont, "leds")

File: 3-Strings/1168-LED/1168.cpp
#include <iostream>

int main() {
    
    int led[10] = {6, 2, 5, 5, 4, 5, 6, 3, 7, 6}, index, sum;
    char input[101];
    std::cin>>index;
    for (int i = 0; i < index; ++i) {
        sum = 0;
        std::cin>>input;
        for (int q = 0; q <= 101; ++q) {
            if (input[q] == '\0')
                break;
            sum += led[((int)input[q])-48];
        }
        std::cout<<sum<<" leds\n";
    }
    return 0;
}